AppConnectSwift/CaptureImage.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  CaptureImage.swift
+//  AppConnectSwift
+//
+//  Created by Suman Sucharit Das on 4/1/16.
+//  Copyright © 2016 Medidata Solutions. All rights reserved.
+//
+
+import UIKit
+
+class CaptureImage: UIViewController, UIImagePickerControllerDelegate, UINavigationControllerDelegate {
+    
+    @IBOutlet weak var imageView: UIImageView!
```

Comment:
Hmmm, I just created it via IBOutlet by dragging and that creates it as weak. Thoughts?